src/meta/src/manager/env.rs

```diff
@@ -73,6 +77,12 @@ pub struct MetaSrvEnv {
     /// system param controller.
     system_params_controller: Option<SystemParamsControllerRef>,
 
+    /// Session param manager
+    session_params_manager: Option<SessionParamsManagerRef>,
+
+    /// Session param controller
+    session_params_controller: Option<SessionParamsControllerRef>,
```

Comment:
Why two `Option` here? Shouldn't it be an `enum`?

Reply:
This seems to be adopted from system parameters. cc @yezizp2012 May I ask if there was any concern for this?

Reply:
No actually, I will open a PR to unify them with `enum`.

Reply:
Done in #16184 .